src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -173,6 +209,38 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
 
    2. In the pop-up, navigate to the [NUbots/NUWebots](https://github.com/NUbots/NUWebots/) repository on your computer and open the `worlds/kid.wbt` world file.
 
+## Running Webots with GameController
+
+1. Find your primary local IP address by running the following.
+
+   ```sh
+   hostname -I
```

Comment:
This didn't work on arch I don't think... @KipHamiltons What did you end up using instead?

Reply:
I think I used `ip addr`. I don't think we should bother adding that to this guide though. Arch users can fend for themselves in this case ;p